src/components/tutor-input.cjsx

```diff
@@ -0,0 +1,74 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+{DateTimePicker} = require 'react-widgets'
+
+TutorInput = React.createClass
+  propTypes:
+    label: React.PropTypes.string.isRequired
+    ref: React.PropTypes.string
+    id: React.PropTypes.string
+    className: React.PropTypes.string
+    onChange: React.PropTypes.func
+    value: React.PropTypes.any
+
+  onChange: ->
+    domNode = @refs[@props.inputRef].getDOMNode()
+    @props.onChange(domNode?.value, domNode)
+
+  render: ->
+    classes = ['form-control']
+    unless @props.default then classes.push('empty')
+    classes.push(@props.class)
+
+    <div className="form-control-wrapper">
+      <input
+        ref={@props.inputRef}
+        id={@props.id}
+        type='text'
+        className={classes.join(' ')}
+        defaultValue={@props.default}
+        onChange={@onChange} />
+      <div className="floating-label">{@props.label}</div>
+    </div>
+
+TutorDateInput = React.createClass
+
+  render: ->
+    classes = ['form-control']
+    unless @props.value then classes.push('empty')
+    <div className="form-control-wrapper">
+      <input type='text' disabled className={classes.join(' ')} />
```

Comment:
This input never gets undisabled...it's just there for the style underneath the date picker.  I couldn't figure out a good way for us to use the paper styles for the date picker otherwise...